java/src/main/java/com/logicalclocks/hsfs/HopsworksConnection.java

```diff
@@ -112,36 +87,6 @@ public FeatureStore getFeatureStore() throws IOException, FeatureStoreException
    * @throws FeatureStoreException FeatureStoreException
    */
   public FeatureStore getFeatureStore(String name) throws IOException, FeatureStoreException {
```

Comment:
`getFeatureStore()` not defined in base and `getFeatureStore(String name)` is missing @Overwrite.